lms/static/sass/shared-v2/_components.scss

```diff
@@ -47,3 +47,48 @@
         }
     }
 }
+
+// TODO: search box should be in the Pattern Library
+.page-header-search {
+    display: inline-block;
+
+    .search-form {
+        display: inline-block;
+    }
+
+    .search-input {
+        display: inline-block;
+        position: relative;
+        vertical-align: middle;
+    }
+
+    .search-field {
+        @include padding($baseline/4, $baseline*1.5, $baseline/4, $baseline/2);
+        transition: all $tmg-f2 ease-in-out;
+        border: 1px solid $gray-l4;
```

Comment:
Are we still using the colour variables, or should we use the Pattern Library approach ```palette(grayscale, white-t)```? Or should there be/are there updated colour variable declarations that use the Pattern Library approach?